charts/karpenter-core/templates/configmap-logging.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-logging
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "karpenter.labels" . | nindent 4 }}
```

Wondering if all of these should be
```suggestion
    {{- include "karpenter-core.labels" . | nindent 4 }}
```

---

It may be a good idea to use karpenter-core in all these configs instead of karpenter. Trying to think about trade offs here. I will have this in a follow up.